src/soundio/sounddevicenetwork.cpp

```diff
@@ -442,10 +442,16 @@ void SoundDeviceNetwork::callbackProcessClkRef() {
 
     if (!m_denormals) {
         m_denormals = true;
+
         // This disables the denormals calculations, to avoid a
         // performance penalty of ~20
         // https://github.com/mixxxdj/mixxx/issues/7747
-#ifdef __SSE__
+
+        // On Emscripten (WebAssembly) denormals-as-zero/flush-as-zero is not
+        // configurable, for discussion and links see
```

Comment:
Please add here a comment about the current situation. Do we suffer from denormals calculations or not?

Reply:
We can probably say that these are not supported, but to assess the actual performance, we'd need probably need audio, which isn't ready yet. Since it's not configurable, there is probably not much point in discussing this as a performance consideration until Emscripten/Wasm add a way to configure it.